app/src/main/java/com/codesoom/assignment/ProductNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package com.codesoom.assignment;
+
+/**
+ * 상품을 찾을 수 없을 때 예외
+ */
```

Comment:
이렇게 표현할 수 있겠죠. 한편 JavaDoc의 첫 설명에는 꼭 마침표를 찍어주세요.
```suggestion
/**
 * 상품을 찾을 수 없는 경우에 던집니다.
 */
```